Makefile

```diff
@@ -186,3 +186,12 @@ fmt-local: ## Reformat files using `go fmt`
 
 imports-local: ## Optimize imports
 	goimports -w -l $$($(FILES_TO_CHECK))
+
+# e2e testing is done here
+.PHONY: e2e-setup
```

Comment:
The comment says `e2e testing` however the target says setup, can you please make it consistent?

Reply:
updated!